### PR TITLE
Benchmarks - Keep BatchNorm as fp32 for pytorch cnn models cast to fp16

### PR DESCRIPTION
**Pull request by jeffdaily**
**Description**
The BatchNorm operator is not numerically stable in fp16.  PyTorch documentation recommends to keep the BN op in fp32 for fp16 AMP models.  Refer to https://pytorch.org/docs/stable/amp.html#ops-that-can-autocast-to-float32.  Preserving BN in fp32 for superbench more accurately reflects real workloads.

### CODE REVIEW

File: superbench/benchmarks/model_benchmarks/pytorch_cnn.py [hunk 1]
```diff
@@ -14,6 +14,12 @@
 from superbench.benchmarks.model_benchmarks.pytorch_base import PytorchBase
 from superbench.benchmarks.model_benchmarks.random_dataset import TorchRandomDataset
 
+def _keep_BatchNorm_as_float(module):
```
Comment (abuccts) [inline superbench/benchmarks/model_benchmarks/pytorch_cnn.py:17]: could you help to fix the lint issue? expected 2 blank lines before and after this function, you can check by `python3 setup.py lint`.

**Review by cp5555 — APPROVED** [review]

**Review by guoshzhao — APPROVED** [review]
